projects/output-and-error-handling/server/client.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	resp, err := http.Get("http://localhost:8080") 
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error making HTTP request: %v\n", err)
+		os.Exit(1)
+	}
+	defer resp.Body.Close()
+	//checking conditions
+	switch resp.StatusCode {
+	case http.StatusOK:
+		fmt.Println("Weather Information:")
+		fmt.Println("Response Status:", resp.Status)
```

Comment:
Does this print out the response from the server describing the weather, or something else?

Reply:
@illicitonion thanks for your comment. the reason for disappearing commits was my mistake of making a PR directly to the main and not making a branch first.
this part of the code checks the response status code and prints the proper message based on the status code so it is not from the server